.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -61,6 +63,54 @@ object WPComPlugins : Project({
 	}
 })
 
+object BuildPlugins: BuildType({
+	id("calypso_WPComPlugins_Build_Plugins")
+	uuid = "8453b8fe-226f-4e91-b5cc-8bdad15e0814"
+	name = "Build WPCom Plugins"
+	description = "Test"
+
+	vcs {
+		root(Settings.WpCalypso)
+		cleanCheckout = true
+	}
+
+	artifactRules = """
+		apps/happy-blocks/release-files => happy-blocks.zip
+		apps/notifications/dist => notifications.zip
+	""".trimIndent()
+
+	steps {
+		mergeTrunk()
+
+		bashNodeScript {
+			name = "Prepare environment"
+			scriptContent = """
+				set -x
+
+				# Update composer
+				composer install
+
+				# Install dependencies
+				yarn
+
+				# Set execution permission for additional scripts.
+				chmod +x .teamcity/scripts/WPComPlugins/
+			"""
+		}
+
+		bashNodeScript {
+			name = "Build artifacts"
+			scriptContent = """
+				# @automattic/wpcom-block-editor needs this variable.
+				export NODE_ENV="development"
```

Comment:
We'll need to make sure this doesn't mess anything up for the other builds -- if so, we could probably put it in a line in wpcom-block editor

Reply:
Good point, I'll incorporate it into the `build-ci` script for `wpcom-block-editor` once I get to that.